docs/system/hardware-metrics.md

```diff
@@ -10,11 +10,14 @@ This document describes instruments and attributes for common hardware level
 metrics in OpenTelemetry. Consider the [general metric semantic conventions](/docs/general/metrics.md#general-metric-semantic-conventions)
 when creating instruments not explicitly defined in the specification.
 
+This document is being converted to specific hardware metrics, parts of this document that have already been
+converted are now located in the [Hardware] (/docs/hardware/README.md) folder and are no longer present in this file.
```

Comment:
```suggestion
converted are now located in the [Common Hardware Metrics](/docs/hardware/common.md) folder and are no longer present in this file.
```

Reply:
This is not correct, it is about all hardware metrics and not only common